src/imageio.cs

```diff
@@ -2356,4 +2368,17 @@ interface CGImageAuxiliaryDataInfo {
 		NSData Data { get; set; }
 		NSDictionary DataDescription { get; set; }
 	}
+
+	[Partial]
+	[Mac (10, 15), iOS (13, 0), TV (13, 0), Watch (6, 0)]
+	interface CGImageAnimationOptions {
+		[Field ("kCGImageAnimationDelayTime")]
+		NSString DelayTime { get; }
```

Comment:
Is it an NSNumber of some kind? Can we use BindAs?

Reply:
Apple docs: `const CFStringRef kCGImageAnimationDelayTime;`

Reply:
From the headers:
```
+/* The value of this key overrides the `delay time' specified by the image
+ * Value is a CFNumber of kCFNumberDoubleType.
+ */
+IMAGEIO_EXTERN const CFStringRef kCGImageAnimationDelayTime  IMAGEIO_AVAILABLE_STARTING(10.15, 13.0);
```
To it is a CFNumber.

Reply:
ok, I'm stupid, is a key, I can't parse english: 'The **value** of this **key**'. To the value if a double.